components/brave_ads/core/internal/account/utility/redeem_confirmation/reward/README.md

```diff
@@ -1,5 +1,11 @@
-# Redeem Reward Confirmation
+# Brave Rewards User Confirmation Token Redemption
 
-Redeem an anonymous [confirmation](https://github.com/brave/brave-browser/wiki/Security-and-privacy-model-for-ad-confirmations), and in return, receive a payment token.
+Return to [confirmation redemption](../../../utility/redeem_confirmation/README.md).
+
+Redeem an anonymous confirmation token and receive a [payment token](../../redeem_payment_tokens/README.md) in return by making a call to https://anonymous.ads.brave.com/v{version}/confirmation/{transactionId}/{credential}.
+
+Here, `{version}` represents the version number of the targeted API, `{transactionId}` is a unique id that is not linkable between confirmation token redemptions, and `{credential}` contains a Base64 URL-encoded signature and the token preimage for the signed confirmation payload.
```

Comment:
> the token preimage for the signed confirmation payload

Can we phrase this in a less technical way? Without going back to the spec, I'm not sure what this is.

Reply:
Change. Thanks